src/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/AppSecondFactor/index.tsx

```diff
@@ -167,7 +168,12 @@ export const AppSecondFactor: React.FC<AppSecondFactorProps> = ({
         flexDirection={["column", "row"]}
       >
         <Box flexBasis="50%">
-          <Text variant={["md", "lg"]}>App Authenticator</Text>
+          <Flex flexDirection={"row"} alignItems="center">
+            <Text variant={["md", "lg"]}>App Authenticator</Text>
+            <Pill mx={4} size="small" variant="gray">
+              Preferred
+            </Pill>
```

Comment:
I don't think we want to use a `Pill` for this info. What does design say? `Sup` with brand blue might be better?

Reply:
>What does design say?

I dropped a message in 🔒 [slack](https://artsy.slack.com/archives/C02BGC5PW/p1699892493938409) before this PR and the response was essentially that we can make the call. 

> Sup with brand blue might be better?

This makes sense and was my first thought, but have only seen it used for counts, so went with this alternative. Happy to switch over to a `Sup` component if you think this is better.

<img width="1026" alt="Screenshot 2023-11-13 at 12 49 24 PM" src="https://github.com/artsy/force/assets/38149304/57481a49-d1ea-422c-beeb-74a4dd758836">

Reply:
👍